Example/FirestoreSample/.gitignore

```diff
@@ -0,0 +1,66 @@
+# Logs
```

Comment:
Is this `.gitignore` providing anything needed not covered by the repo level `.gitignore`?

 We've had a policy of keeping all `.gitignore`s  at the top of the repo. It might be ok to make an exception for Samples, but I'm not sure it's needed here.

Reply:
This one is generated by `firebase init`, I believe. We can remove it, and add any patterns that are relevant to the top level ignore file.